src/store/migrations.test.js

```diff
@@ -13,6 +13,10 @@ jest.mock('../services/platform.service', () => ({
   windowsHomeDir: 'test',
 }));
 
+jest.mock('path', () => ({
+  join: () => 'test/guppy-projects',
+}));
+
 const ENGINE_KEY = 'test-key';
```

Comment:
`ENGINE_KEY` seems unused as ESLint reported.
Should we remove it?

Reply:
Yeah, I believe I removed this in one of my PRs. Feel free to remove as well :)